XCUITestHelpers.xcodeproj/xcshareddata/xcschemes/PlaygroundApp.xcscheme

```diff
@@ -26,7 +26,18 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "YES"
+      codeCoverageEnabled = "YES"
+      onlyGenerateCoverageForSpecifiedTargets = "YES">
+      <CodeCoverageTargets>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "3FF1423B266DB44900138163"
+            BuildableName = "XCUITestHelpers.framework"
+            BlueprintName = "XCUITestHelpers"
+            ReferencedContainer = "container:XCUITestHelpers.xcodeproj">
+         </BuildableReference>
+      </CodeCoverageTargets>
```

Comment:
This enables gathering test coverage, which I found useful to check I added tests for all the new utils.

You can see the coverage from here:

![image](https://user-images.githubusercontent.com/1218433/129850073-adab9d77-bb86-4449-957b-f01a6fa2d5a3.png)

Reply:
Nice! It's the first time I see coverage "in action" and being actually informative (not saying I tried too hard before).